src/components/bar.cpp

```diff
@@ -818,9 +810,27 @@ void bar::handle(const evt::button_press& evt) {
     }
   };
 
+  bool has_dblclick = false;
+  switch (btn) {
+    case mousebtn::LEFT:
+      has_dblclick = m_dblclicks.at(mousebtn::DOUBLE_LEFT) or
+                     m_action_ctxt->has_action(mousebtn::DOUBLE_LEFT, pos) != tags::NO_ACTION;
+      break;
+    case mousebtn::MIDDLE:
+      has_dblclick = m_dblclicks.at(mousebtn::DOUBLE_MIDDLE) or
+                     m_action_ctxt->has_action(mousebtn::DOUBLE_MIDDLE, pos) != tags::NO_ACTION;
+      break;
+    case mousebtn::RIGHT:
+      has_dblclick = m_dblclicks.at(mousebtn::DOUBLE_RIGHT) or
+                     m_action_ctxt->has_action(mousebtn::DOUBLE_RIGHT, pos) != tags::NO_ACTION;
+      break;
+    default:
+      break;
+  }
```

Comment:
Could you maybe simply this using the `mousebtn_get_double` function? That way, we don't need the whole switch statement here. Also please use `||` instead of `or`.

Reply:
Thanks for the feedback, I applied the changes you suggested